mathesar/api/ui/permissions/database_role.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import Q
+from rest_access_policy import AccessPolicy
+
+from mathesar.models.base import Database
+from mathesar.models.users import DatabaseRole, Role
+
+
+class DatabaseRoleAccessPolicy(AccessPolicy):
+    statements = [
+        # Listing and retrieving a database role is allowed for everyone.
+        # We cannot restrict access for creating a Database Role object here because database for which the role is created can be known only by inspecting the request body
```

Comment:
Please wrap the comments so the lines are shorter.

Reply:
Also please ensure correct grammar (I see randomly capitalized words and no spaces between words and parentheses, for example)